### PR TITLE
fix: Allow passing an event name to the `emitted` method

### PR DESCRIPTION
**Pull request by dobromir-hristov**

### CODE REVIEW

File: src/vue-wrapper.ts [hunk 1]
```diff
@@ -62,10 +62,10 @@ export class VueWrapper<T extends ComponentPublicInstance> {
     return true
   }
 
-  emitted(): Record<string, unknown[]> {
+  emitted(eventName?: string): Record<string, unknown[]> | undefined {
```
Comment (pikax) [inline src/vue-wrapper.ts:65]:
```suggestion
  emitted(): Record<string, unknown[]> | undefined
  emitted<T = unknown>(eventName: string): T[] | undefined
  emitted(
    eventName?: string
  ): Record<string, unknown[]> | unknown[] | undefined {
```

Comment (lmiller1990) [reply src/vue-wrapper.ts:65]: @dobromir-hristov when you get a chance can you add this and make sure it's compiling? And we can merge :) no rush!

Comment (lmiller1990) [reply src/vue-wrapper.ts]: Ok I did it 👍

**Comment by lmiller1990** [comment]
I added the types and tsd tests - will merge when CI finishes.

**Comment by dobromir-hristov** [comment]
Yep Thanks, been away as you can see :D

**Comment by lmiller1990** [comment]
I really like having a bunch of us working on this. That way if someone is busy, it's completely fine 👍

**Pull request merged**